lib/cmd/common/admin-policies.js

```diff
@@ -165,32 +170,37 @@ function update(guid, policyId, type, config, checkUserExists, checkUserApproved
     "checkUserExists" : checkUserExists || false,
     "checkUserApproved" : checkUserApproved || false
   };
+  var url = fhreq.getFeedHenryUrl();
+  var msg = i18n._("Error updating policy: ");
 
-  common.doApiCall(fhreq.getFeedHenryUrl(), "/box/srv/1.1/admin/authpolicy/update", payload, "Error updating policy: ", function(err, data){
+  common.doApiCall(url, "/box/srv/1.1/admin/authpolicy/update", payload, msg, function(err, data){
     if(err) return cb(err);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)